### DISCUSSION TITLE
Tailwind animation on field validation

### DISCUSSION THREAD

**Discussion by gedi-t** [tailwindlabs/tailwindcss#3371] [Help]
Hey all!

I got a question, I am trying to create field validation error message with tailwind and would like to use Tailwind Animations such as animate-ping or animate-pulse, to animate the borders of the field. 

I was trying  animate-ping:border-b-2 and other ways but counting make it work, can you advice me if this is possible and if so how to achieve that. 

1000 color class is a custom color code just in case you have questions

```
<div class="relative **animate-ping:border-b-2** focus-within:border-red-1000">
        <input type="password" name="password" placeholder=" " class="app-field-input" />
        <label for="password" class="app-field-label">Validation</label>
      </div>
```

![image](https://user-images.githubusercontent.com/19917883/104810445-df01b580-57f4-11eb-8217-b343c4872c55.png)
This is my css for the in appp fields

```

/*In app fields*/

input:focus-within ~ label,
input:not(:placeholder-shown) ~ label {
  @apply transform scale-90 -translate-y-7 text-blue-1000;
}

input:focus-within ~ label {
  @apply text-blue-1000;
}

.app-field-div {
  @apply relative border-b-2 focus-within:border-blue-1000;
}
.app-field-input {
  @apply block w-full appearance-none focus:outline-none bg-transparent;
}

.app-field-label {
  @apply absolute top-0 -z-1 text-lg duration-300 origin-0 text-gray-1000;
}

/*END In app fields*/```

**Comment by simonswiss** [comment]
Hey!

Not sure I understand exactly what you're trying to do, but could. you please try to create a simple Tailwind Play with your setup?

You can't animate the border of an element with a `pulse` animation, but you could have an element (`div`) _behind_ your form element, which is the one you animate to give the impression the border of the form is animated 👍

**Reply by ferpmdev** [reply]
Sorry, but the animation not work in div because in the input, this just start with focus
